.travis.yml

```diff
@@ -1,14 +1,11 @@
 language: go
 go:
   - 1.5.1
-branches:
-  only:
-    - master
 services:
   - docker
 sudo: required
 script:
-  - make bootstrap test
+  - make test docker-build
```

Comment:
Oops, nvm. Read up and saw that `bootstrap` is a noop and `build` doesn't exist...